src/PKSim.Core/Model/ParameterAlternativeGroup.cs

```diff
@@ -19,20 +19,19 @@ public class ParameterAlternativeGroup : Container
 
       public virtual void RemoveAlternative(ParameterAlternative parameterAlternative) => RemoveChild(parameterAlternative);
 
-      public virtual ParameterAlternative DefaultAlternative => AllAlternatives.FirstOrDefault(x => x.IsDefault);
+      /// <summary>
+      ///    Returns the default alternative if defined otherwise the first alternative, if any is available
+      /// </summary>
+      public virtual ParameterAlternative DefaultAlternative => AllAlternatives.FirstOrDefault(x => x.IsDefault) ?? AllAlternatives.FirstOrDefault();
```

Comment:
I changed that slightly to return the first with default or the first item. That way we should always have one